_posts/2019-08-02-Rates-Rebates-Recommendations-Report.md

```diff
@@ -0,0 +1,276 @@
+---
+title:Recommendations Report
+date: 2019-08-02
+authors:
+  - Service Innovation Lab
+tags:
+  - blog post
+  - open source
+  - rates rebates
+  - Te Whakamāmā i ngā Reiti
+excerpt: "Following a successful Alpha phase in May and September 2018, the Rates Rebate beta trial was run in June and July 2019 by a team with members from Service Delivery and Operations team within DIA, Government Information Services, and the Service Innovation Lab. This was in collaboration with Tauranga City Council, Hutt City Council, and Kāpiti Coast District Council"
+---
+## Executive Summary
+
+A Rates Rebate is a subsidy (currently up to $640) for low-income homeowners on the cost of their residential rates. The subsidy is delivered by Local Councils and is administered by Central Government (currently the Department of Internal Affairs (DIA)). The entitlement value is calculated by the cost of the rates, applicants income, and number of dependants. The maximum subsidy amount is reviewed yearly.
```

Comment:
"The maximum subsidy amount is reviewed yearly."
By who? can we specify to make the sentence clearer?